src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static ColorSupplier colorSupplier;
+
+    public Lottery(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public static Ball getRandomBall() {
```

Comment:
Please do not use static methods
```suggestion
    public Ball getRandomBall() {
```